### PR TITLE
Create scribe component

### PR DESCRIPTION
**Pull request by ShaunYearStrong**
picture 83

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +44,12 @@
     "react-redux": "^5.0.2",
     "react-router": "^3.0.2",
     "redux": "^3.6.0",
-    "redux-thunk": "^2.2.0"
+    "redux-thunk": "^2.2.0",
+    "scribe": "guardian/scribe#fcf4e16",
```
Comment (ShaunYearStrong) [inline package.json]: This will need changing when there's a new release of scribe

File: public/js/components/ReactApp.js [hunk 2]
```diff
@@ -1,41 +1,37 @@
...
-import FormFieldTextInput from './FormFields/FormFieldTextInput';
-import FormFieldSelectBox from './FormFields/FormFieldSelectBox';
-import FormFieldRadioButtons from './FormFields/FormFieldRadioButtons';
+import ScribeEditor from './ScribeEditor/ScribeEditor';
 
 export default class ReactApp extends React.Component {
 
-  updateCheckbox = () => {
-    console.log("Updated");
+  updateDescription = (newContent) => {
```
Comment (ShaunYearStrong) [inline public/js/components/ReactApp.js]: just an example.

**Comment by ShaunYearStrong** [comment]
I'm going to remove noting for now. The broken build is caused by it having a dependency on bower.

**Comment by clloyd** [comment]
Is this missing styles?

**Comment by ShaunYearStrong** [comment]
@clloyd Kind of, it doesn't have any buuuut I didn't want to make assumptions about the usage of scribe in here. I think we're coming to a point where we're going to have to decide on some kind of page structure here

**Comment by clloyd** [comment]
The general component doesnt need to have styles, but we should pass in a relevant class and style that in the example

**Pull request merged**